monitorrent/engine.py

```diff
@@ -588,16 +595,19 @@ def _execute(self, ids=None):
         caught_exception = None
         self.is_executing = True
         try:
+            log.info("Starting execute", time=str(datetime.now()))
             self.logger.started(datetime.now(pytz.utc))
             engine = Engine(self.logger, self.settings_manager, self.trackers_manager,
                             self.clients_manager, self.notifier_manager)
             engine.execute(ids)
         except:
             caught_exception = sys.exc_info()[0]
+            log.error("An error has occurred during execute", exception=str(caught_exception))
```

Comment:
We store it for next usage, not for printing right now.
And from docs:
> Process event and call logging.Logger.error() with the result, after setting exc_info to True.

That was what I'm asked for, thx